wpcrm/quickstart.md

```diff
@@ -31,19 +31,44 @@ For more information about solutions please refer to [Dynamics 365 Solutions doc
 
 ![Create a new form using Solution Explorer](/img/wpcrm/createform-solutionexplorer.png)
 
-In Solution Explorer navigate to **Components > Entities** section and choose the desired entity, for example, *Contacts*. Expand the selected entity node and select *Forms*. Click the *New* button and select the desired form type.
+In Solution Explorer navigate to **Components > Entities** section and choose the desired entity, for example, *Contact*. Expand the selected entity node and select *Forms*. Click the *New* button and select the desired form type.
 For more information about form types please see [Create and design forms](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/customize/create-design-forms).
 
 ![Form editor](/img/wpcrm/createform-editor.png)
 
 Now you can create and save the new form using the form editor.
 For more information about the form editor please see [Form editor documentation](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/customize/form-editor-user-interface-legacy).
 
-Please don't forget to publish your form.
+Please don't forget to specify name for your custom form and publish it.
 
-### Putting a form on the page
+### Adding a form to a page
 
-TBD.
+Create a new page in your WordPress administration panel by navigating to **Pages &gt; Add new** or open an existing one from **Pages &gt; All pages** menu section. Don't forget to enter page title if it's a new page.
+
+Switch content editor mode to plain text by clicking on the *Text* tab to be able to insert code in content field. Finally insert this code to *Content* text field:
+
+{% raw %}
+```twig
+[msdyncrm_twig]
+{% form entity="contact" name="WordPress Example Form" mode="create" %}{% endform %}
+[/msdyncrm_twig]
+```
+{% endraw %}
+
+Where
```

Comment:
Please remove this "where" section. The parameters used here are very self-evident. Maybe leave the `mode`. Say something like "In `create` mode the form will create a new record in CRM".